src/mainwindow.cpp

```diff
@@ -234,6 +236,23 @@ void MainWindow::setup_ActionsMenu_Actions()
     setup_Action(MOVE_RIGHT, new QAction(tr("Move Tab &Right"), settingOwner),
                  MOVE_RIGHT_SHORTCUT, consoleTabulator, SLOT(moveRight()), menu_Actions, data);
 
+    menu_Actions->addSeparator();
+    data.setValue(checkHasIndexedTab);
+
+    const QString textBase = tr("Tab");
+    QSignalMapper *signalMapper = new QSignalMapper(this);
```

Comment:
From https://doc.qt.io/qt-5/qsignalmapper.html:

> This class is obsolete. It is provided to keep old source code working. We strongly advise against using it in new code.

Reply:
Didn't realize it had been deprecated. I dropped it in the last patch.